.github/workflows/first-interaction.yml

```diff
@@ -0,0 +1,29 @@
+name: "Welcome New Contributors"
+
+on:
+  pull_request:
+    types: opened
+  issues:
+    types: opened
+
+permissions:
+  pull-requests: write
+  issues: write
+
+jobs:
+  welcome-message:
+    name: Welcome Contributors
+    runs-on: ubuntu-latest
+    timeout-minutes: 10
+    if: github.event.action == 'opened'
+    steps:
+      - uses: actions/first-interaction@v1
+        with:
+          repo-token: ${{ secrets.GITHUB_TOKEN }}
+          pr-message: |
+            Thank you for creating your first Pull Request and for being a part of the open signing revolution! 💚🚀
+            <br /> Feel free to hop into our community in [Discord](https://documen.so/discord)
+          issue-message: |
+            Thank you for opening your first issue and for being a part of the open signing revolution! 
+            <br /> One of our team members will review it and get back to you as soon as it possible 💚
+            <br /> Meanwhile, please feel free to hop into our community in [Discord](https://documen.so/discord)
```

Comment:
For this we want to remind people of the contribution guidelines, we should avoid too much automation around first interactions since I feel it's better for the team to do those so we're genuine 😅

Reply:
Gotcha, would love to know your opinion on what needs to be removed. Should we have reminders for contribution guidelines only?